plugin/trino-hive/src/main/java/io/trino/plugin/hive/gcs/GcsConfigurationProvider.java

```diff
@@ -31,7 +31,7 @@
     @Override
     public void updateConfiguration(Configuration configuration, HdfsContext context, URI uri)
     {
-        if (!uri.getScheme().equals(GoogleCloudStorageFileSystem.SCHEME)) {
```

Comment:
nit: Can we do a static import as a prep. commit and then maybe update the library ?